App/Services/UserNotifier.cs

```
@@ -20,17 +21,26 @@ public interface IUserNotifier : INotificationHandler, IAsyncDisposable
     public void UnregisterHandler(string name);
 
     public Task ShowErrorNotification(string title, string message, CancellationToken ct = default);
-    public Task ShowActionNotification(string title, string message, string handlerName, IDictionary<string, string>? args = null, CancellationToken ct = default);
+    public Task ShowActionNotification(string title, string message, string? handlerName, IDictionary<string, string>? args = null, CancellationToken ct = default);
 }
 
-public class UserNotifier(ILogger<UserNotifier> logger, IDispatcherQueueManager dispatcherQueueManager) : IUserNotifier
+public class UserNotifier : IUserNotifier
 {
     private const string CoderNotificationHandler = "CoderNotificationHandler";
 
     private readonly AppNotificationManager _notificationManager = AppNotificationManager.Default;
+    private readonly ILogger<UserNotifier> _logger;
+    private readonly IDispatcherQueueManager _dispatcherQueueManager;
 
     private ConcurrentDictionary<string, INotificationHandler> Handlers { get; } = new();
 
+    public UserNotifier(ILogger<UserNotifier> logger, IDispatcherQueueManager dispatcherQueueManager)
+    {
+        _logger = logger;
+        _dispatcherQueueManager = dispatcherQueueManager;
+        Handlers.TryAdd(nameof(DefaultNotificationHandler), new DefaultNotificationHandler());
```

This should probably fail rather than silently do nothing